packages/slice-machine/src/components/BlankSlate/BlankSlate.css.ts

```diff
@@ -23,6 +23,19 @@ export const root = style([
   { maxWidth: 498, minHeight: 400 },
 ]);
 
+export const hasBackground = style([
```

Comment:
`hasBackground` sounds more like a condition than a className. Maybe name it `withBackground` or `cover`?

Reply:
You're right, will change that :)

Reply:
Done 👍